rich-default/Dockerfile

```diff
@@ -32,103 +32,11 @@ RUN apt-get update && \
     texlive-full && \ 
     rm -rf /var/lib/apt/lists/*
 
-#RUN apt-get update && \
-#    apt-get install -y --no-install-recommends ffmpeg \
-#    fonts-dejavu \
-#    gfortran \
-#    gcc \
-#    octave \
-#    octave-control \
-#    octave-image \
-#    octave-io \
-#    octave-optim \
-#    octave-signal\
-#    octave-statistics && \
-#    rm -rf /var/lib/apt/lists/*
```

Comment:
Double check this is supposed to be commented out.

Reply:
Yes since we install `octave-*`